### ISSUE TITLE
Auto upload setting adds "- Auto upload" to selected existing directory name

### ISSUE THREAD

**Issue by pokaalinkanssayohon** [nextcloud/ios#2227]
### Steps to reproduce

1. Settings --> Auto upload
2. Select the "Auto upload" folder
3.

### Expected behaviour

Selected folder will be used for auto uploads

### Actual behaviour

Renaming existing selected folder by adding "- Auto upload". Auto upload doesn't work at all.

### Screenshots

If applicable, add a screenshot showing the issue.

### Logs

```
If applicable, you can post the iOS app or server logs (removing any sensitive information).
```

### Environment data

**iOS version:** e.g. iOS 16.1

**Nextcloud iOS app version: iOS 4.5.3.0

**Server operating system: Nexcloudpi (v1.50.3)
![IMG_1486](https://user-images.githubusercontent.com/34340500/198834691-699995ee-9d3a-42e5-a89f-29b1f6481565.jpeg)

**Web server:** Apache, nginx

**Database:**

**PHP version:**

**Nextcloud version:** see Nextcloud admin page

**Comment by pokaalinkanssayohon** [comment]
[communication.log.txt](https://github.com/nextcloud/ios/files/9893806/communication.log.txt)

**Comment by marinofaggiana** [comment]
Hi @pokaalinkanssayohon I don't understand the issue.

**Comment by pokaalinkanssayohon** [comment]
This is duplicated issue of #2225 . I assumed, that broken auto upload was related to added “- Auto upload” in iOS app UI and maybe broken some symlink because of that. If this is not the issue at all, this can be closed and let’s continue with issue #2225
